### DISCUSSION TITLE
MEDIANS

### DISCUSSION THREAD

**Discussion by SantiagoOsorio3** [IIC2133-PUC/2024-2#90] [Tarea 1] [answered]
Hola, como estan ? queria preguntar a que se refiere la mejor y peor mediana, porque yo habia entendido era de distintos sectores, pero no me queda claro esto:

Si la mejor mediana y la peor mediana pertenecen al mismo sector, se considera que el sector con menor ID es el mejor, y el sector con mayor ID es el peor.

**Comment by 4n1t4** [comment]
Error de redacción. La idea principal es:

* Cuando ocurre un caso borde de empate de sectores, por ejemplo existen solamente 2 sectores con la misma mediana. En este caso el desempate se da por ID del sector

* cuando solo hay un sector este es el mismo para tanto la mejor como peor mediana.

**Comment by santiago3c** [comment]
y que pasaría en el siguiente caso?:
sector: 0 mediana: 30
sector: 1 mediana: 30
sector: 2 mediana: 40
sector: 3 mediana: 50
sector: 4 mediana: 50

**Comment by 4n1t4** [comment, marked as answer]
ANUNCIO!

Error de enunciado, corregiremos el enunciado para que diga que en caso de empate siempre se prefiere el de menor ID. Por ende , si tenemos el caso de que mencionas antes:  
se selecciona el sector 0 y 3.

**Comment by santiago3c** [comment]
recibido
